src/database/migrations/20190827125917-create-user.js

```diff
@@ -13,7 +13,7 @@ module.exports = {
         notNull: true,
       },
     },
-    userName: {
+    username: {
```

Comment:
I don't think you should remove the camel case here

Reply:
Any reason for this? If you would like to update columns names, that should be done with another migration file.